sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -206,6 +208,11 @@ export class ConfirmationOfEnrollmentController {
       );
     }
 
+    //TODO: Document Number should be Updated in the same transaction when COE status changes
```

Comment:
Sequentially COE confirmation should happen first and then document number should be generated. The order doesn't matter if it is under same transaction.
I see a todo here. Until the todo is implemented, this sequence should be followed IMO.

Reply:
I had a chat with @andrewsignori-aot before moving the line of code up. There will be more work coming down the line   which will also involve some refactoring.